lib/lib.misc.php

```diff
@@ -1348,14 +1388,28 @@ function eventlog($type, $dataid, $action, $cid = null, $json = null, $id = null
 		return;
 	}
 
+	$idsCombined = implode(',', $ids);
+
+	// We should pass multiple ID's if instructed to do so
+	$multiple = count($dataids) > 1 || count($ids) > 1;
+
 	// Generate JSON content if not set, for deletes this is only the ID.
 	if ( $action === 'delete' ) {
-		$json = "{\"id\":\"$id\"}";
+		$json = array_values(array_map(function($id) {
```

Comment:
my PHP foo is weak, what are you trying to do here?

Reply:
It maps every `$id` to `['id'=>$id]` and then makes sure the array is 0-indexed.